### ISSUE TITLE
Crashing under iOS 8 / Xcode 6 / iPhone 6

### ISSUE THREAD

**Issue by dwieringa** [rickyzhang82/tethering#12]
I've installed this on my iPhone 6, but the app crashes frequently.  I haven't been able to get it to work yet.  It looks like you've built against iOS 8.  Have you been using this on an iPhone 6?

For what it is worth, I'm trying to use it with a HooToo TripMate Nano Travel Router HT-TM02

**Comment by rickyzhang82** [comment]
No, I haven't. Mine is 5s. You can post your crash log here. 

If you want to debug yourself, enable NSLogger in SocksProxy_Prefix.pch

**Comment by dwieringa** [comment]
In the meantime, I got it working on an iPhone 4S (iOS 7.1.2).  Unfortunately I've only been able to get it to work using an Ad Hoc network created on my Mac.  When I try to use a network created by the HooToo HT-TM02 router, it does not work (even though all settings are the same). *[by the way, to get it working on the 4S with 3.5" screen I had to move the Start button]*

I just switched back to the iPhone 6 (iOS 8.1) and I have it working now using the Ad Hoc network setup.  It's using the travel router where I haven't gotten it to work and I see the crashes.  I'll try to investigate further later.

**Comment by dwieringa** [comment]
In the meantime, I got it working on an iPhone 4S (iOS 7.1.2) using an Ad Hoc network from my Mac. *[by the way, to get it working on the 4S with 3.5" screen I had to move the Start button]*

I then switched back to the iPhone 6 (iOS 8.1) and got it working there also via ad hoc.  After further tinkering I have it working using a network from my HooToo HT-TM02.

Regardless, it does crash under certain circumstances.  If time permits, I'll document my successful setup and try to reproduce the crashes.
